### PR TITLE
Features/api/checkpoint

### PR DESCRIPTION
**Pull request by vpulgar**
Implementation curriculum and learning unit get endpoints.

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -1,25 +1,12 @@
...
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 5
+# Offense count: 1
 # This cop supports safe autocorrection (--autocorrect).
 # Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, AllowedPatterns, IgnoredPatterns.
 # URISchemes: http, https
-Layout/LineLength:
-  Max: 93
```
Comment (isavega) [inline .rubocop_todo.yml:14]: JAJAJJA

File: app/controllers/api_resources_controller.rb [hunk 2]
```diff
@@ -1,2 +1,23 @@
...
+  end
+
+  def show
+    resource = Curriculum
+               .find(params[:curriculum_id])
+               .learning_units
+               .find(params[:learning_unit_id])
+               .resources
+               .find(params[:resource_id])
+    # Para tener un path mas lindo
```
Comment (isavega) [inline app/controllers/api_resources_controller.rb:18]: Comentarios

File: app/controllers/api_curriculums_controller.rb [hunk 3]
```diff
@@ -1,6 +1,11 @@
 class ApiCurriculumsController < ApiApplicationController
   def index
-    curriculums = Curriculum.all()
+    curriculums = Curriculum.all
```
[the remark this answers is not in the document]
Comment (vpulgar) [reply app/controllers/api_curriculums_controller.rb:3]: Si no me equivoco es lo mismo. Lo teste y no se ven diferencias

[the remark this answers is not in the document]
Comment (cecyramallo) [reply app/controllers/api_curriculums_controller.rb:3]: Rubocop marks this as an error.

**Pull request merged**